lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -8,26 +8,27 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
             <div class="wrapper-copy-and-actions">
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
-                %if badge:
-                    <p class="sr-only">${_("Share on:")}</p>
+                    <p class="sr-only">Take this with you:</p>
```

Comment:
If its not going to be displayed visually, can we make this a Heading? Makes it that much easier to find for a SR user.  Also, do we want to 118n it?  We usually do.